packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -446,6 +470,24 @@ class _ExpansionPanelListState extends State<ExpansionPanelList> {
         context,
         _isChildExpanded(index),
       );
+      final Widget expansionIndicator = child.expansionIndicator != null
+          ? InkResponse(
+        onTap: !child.canTapOnHeader
+            ? () => _handlePressed(_isChildExpanded(index), index)
+            : null,
+        child: child
```

Comment:
Formatting of this block here is a little off. This would be clearer:
```
child: child.expansionIndicator(
  context,
  _isChildExpanded(index),
),
```